spec.bs

```diff
@@ -2256,6 +2257,19 @@ The network revocation mechanism requires the following monkeypatches to the [[F
   (<a href="https://github.com/WICG/fenced-frame/issues/191">WICG/fenced-frame#191</a>)
 </div>
 
+<div algorithm=resolve-an-origin-patch>
+  Modify the [=resolve an origin=] algorithm to include an extra parameter: an [=url=] |url|.
+  
+  Insert these steps immediately after step 2:
+  
+  1. Let |nonce| be |key|'s [=network partition key/nonce=].
+  
+  1. If the user agent's [=network revocation exemption map=][|nonce|] [=map/exists=] and |url| does
+     not [=set/exist=] in it, then:
```

Comment:
I would make this more clear and verbose by saying if NREM[nonce] exists, but NREM[nonce][url] does not exist, then....